src/entities/Publication.ts

```diff
@@ -8,31 +8,43 @@ export class Publication {
 
   @Field()
   @Property()
-  bio: string;
+  backgroundImageURL: string;
 
   @Field()
   @Property()
-  rssURL: string;
+  bio: string;
 
   @Field()
   @Property()
-  imageURL: string;
+  bioShort: string;
 
   @Field()
   @Property()
   name: string;
 
   @Field()
-  @Property({ unique: true })
-  websiteURL: string;
+  @Property()
+  profileImageURL: string;
 
   @Field()
   @Property({ unique: true })
   rssName: string;
 
+  @Field()
+  @Property()
+  rssURL: string;
+
+  @Field()
+  @Property()
+  slug: string;
```

Comment:
not sure if we need should place `unique: true` everywhere, but what are your thoughts on enforcing these as unique vs. not?

Reply:
Adding `unique: true` adds some extra overhead when adding a publication to the list. They should be unique, but since all of this content is generated by us with the hardcoded `publications.json` I don't think it's necessary. I did make the slug unique though, because that's just the definition of slugs and at least one field needs to be unique.

Reply:
it says only the rssName is unique? not the slug